Hearthstone Deck Tracker/Core.cs

```diff
@@ -118,8 +118,10 @@ public static void Initialize()
 			MainWindow.Options.OptionsTrackerPlugins.Load();
 			PluginManager.Instance.StartUpdateAsync();
 
-			UpdateOverlayAsync();
-			NewsUpdater.UpdateAsync();
+            if (Config.Instance.ShowNewsBar)
```

Comment:
Checking this removed the ability for me to remotely enable the news bar again, which might be useful is some cases.